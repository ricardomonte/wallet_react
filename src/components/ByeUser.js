import React, {useState} from 'react';
import { Headline3} from '@material/react-typography';
import '@material/react-typography/dist/typography.css';
import { Redirect } from 'react-router-dom';

const ByeUser = () => {
  const [goHome, setGoHome] = useState(false);

  const handleReturn = () => {
    setGoHome(!goHome)
  }
  return(
    <div className='loading_errors'>
      <Headline3>See you soon</Headline3>
      <button type='button' onClick={handleReturn}>Return Home</button>
      {goHome && <Redirect to='/' />}
    </div>
  );
}

export default ByeUser;