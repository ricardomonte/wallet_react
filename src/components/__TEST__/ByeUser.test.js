import React from 'react';
import '@testing-library/jest-dom';
import { render,screen } from '@testing-library/react';
import ByeUser from '../ByeUser';

test('Content from ByeUser matches snapshot', () => {
  const content = render(<ByeUser />);
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('See you soon')).toBeInTheDocument();
});