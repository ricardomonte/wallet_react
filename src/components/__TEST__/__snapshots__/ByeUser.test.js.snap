// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content from ByeUser matches snapshot 1`] = `
<div>
  <div
    class="loading_errors"
  >
    <h3
      class="mdc-typography mdc-typography--headline3"
    >
      See you soon
    </h3>
    <button
      type="button"
    >
      Return Home
    </button>
  </div>
</div>
`;
